### DISCUSSION TITLE
I need more help with the account blocked at registration.

### DISCUSSION THREAD

**Discussion by jhjhfjdhfjdhfjd** [vercel/community#3753] [Account Issue] [answered]
### Question

Basically its almost been 3 days since I sent out my reply to the registration email so I can get my account unblocked and they just didn't respond (something deleted my last discussion about this seems fishy vercel)

Please help.

**Comment by amyegan** [comment, marked as answer]
Hey @jhjhfjdhfjdhfjd. Your previous post was closed as a duplicate of the original. Nothing was deleted as far as I am aware. This post is also being closed as a duplicate of the original post https://github.com/vercel/community/discussions/3710. 

I am sorry you're having account trouble. As stated previously, I cannot discuss account information here due to safety and security concerns for your personal data. Please follow up with your support case if you have further questions. 

If there is anything else you would like to share with me, please continue the conversation in https://github.com/vercel/community/discussions/3710. This will help me keep track of the conversation more easily. 🙏
